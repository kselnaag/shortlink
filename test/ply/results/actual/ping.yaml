ping:  # 22.10.2023, 20:04:10:160
  request:
    url: http://localhost:8080/check/ping
    method: GET
    headers: {}
  response:  # 45 ms
    status:
      code: 200
      message: OK
    headers: {}
    body: |-
      {
        "Body": "pong",
        "IsResp": true,
        "Mode": "check"
      }
