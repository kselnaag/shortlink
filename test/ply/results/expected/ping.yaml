ping:
  request:
    url: ${baseUrl}/check/ping
    method: GET
    headers: {}
  response:
    status:
      code: 200
      message: OK
    headers: {}
    body: |-
      {
        "Body": "pong",
        "IsResp": true,
        "Mode": "check"
      }
